services/blockchain-app-registry/tests/unit/mysqlKVStore.test.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { formatValue } = require('../../shared/database/mysqlKVStore');
+
+describe('Unit tests for mysqlKVStore', () => {
+	describe('Unit tests for formatValue', () => {
+		it('Should return boolean when type is boolean', () => {
+			const response = formatValue(123, 'boolean');
+			expect(response).toBe(true);
+			expect(typeof response).toBe('boolean');
+		});
```

Comment:
This test can be updated to: 
```suggestion
		it('Should return boolean when type is boolean', () => {
			const response1 = formatValue(0, 'boolean');
			expect(response1).toBe(false);
			expect(typeof response1).toBe('boolean');
			
			const response2 = formatValue(1, 'boolean');
			expect(response2).toBe(true);
			expect(typeof response2).toBe('boolean');
		});
```

Reply:
Please ensure that the tests also reflect the original usage. Please add scenarios where we pass boolean values as well.